src/fev/metrics.py

```diff
@@ -244,12 +287,15 @@ def compute(
         seasonal_error = _abs_seasonal_error_per_item(
             past_data=past_data, seasonality=seasonality, target_column=target_column
         )
+        seasonal_error = np.clip(seasonal_error, self.min_error, None)
         return self._safemean(ql_per_time_step / seasonal_error[:, None])
 
 
 class WQL(Metric):
```

Comment:
Let's also add eps / min_scale to `WAPE` and `WQL` metrics since they also suffer from the division by zero?